src/main/java/com/google/cloud/resourcemanager/spi/v1beta1/ResourceManagerRpc.java

```diff
@@ -57,6 +59,29 @@ Integer getInt(Map<Option, ?> options) {
     }
   }
 
+  class ListResult<T> {
+
+    private final Iterable<T> results;
+    private final String pageToken;
+
+    ListResult(String pageToken, Iterable<T> results) {
+      this.results = ImmutableList.copyOf(results);
+      this.pageToken = pageToken;
+    }
+
+    public static <T> ListResult<T> of(String pageToken, Iterable<T> list) {
```

Comment:
don't need to be public

Reply:
this need to be public for to use `of` method out side this package. if its not then we cannot accessed it from outside this package

Reply:
But the class isn't public so it can't be accessed from outside this package.

Reply:
done